changelogs/unreleased/8252-remove-deploy-status.yml

```
@@ -0,0 +1,19 @@
+description: Remove `deployed` and `result from configuration model
+issue-nr: 8252
+change-type: major
+destination-branches: [master]
+sections:
+  upgrade-note: >
+    The fields `deployed` and `result` have been removed from all API endpoint and tools that report on versions.
```

Not 100% sure but this is probably better if meant to be readable from the changelog so that it renders as typewriter font instead of italics: (same in the table)
```suggestion
    The fields ``deployed`` and ``result`` have been removed from all API endpoint and tools that report on versions.
```